connectors/rocketmq-connect-elasticsearch/pom.xml

```diff
@@ -223,7 +223,7 @@
         <dependency>
             <groupId>org.elasticsearch.client</groupId>
             <artifactId>elasticsearch-rest-high-level-client</artifactId>
-            <version>7.6.2</version>
+            <version>7.14.0</version>
```

Comment:
Is ES 8.x supported?

Reply:
I will try to upgrade the version to 8.x to see if there are any conflicts

Reply:
The 8.x version of es supports at least jdk17, so we won't upgrade to 8.x yet